projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -174,16 +174,66 @@
 			el.dispatchEvent( e );
 		}
 
-		function scrollToElement( el ) {
-			if ( ! el || typeof el.scrollIntoView !== 'function' ) {
+		// From: https://easings.net/#easeInOutQuad
+		function easeInOutQuad( num ) {
+			return num < 0.5 ? 2 * num * num : 1 - Math.pow( -2 * num + 2, 2 ) / 2;
+		}
+
+		function scrollToElement( el, container, callback ) {
+			if ( ! el || ! container ) {
+				if ( callback ) {
+					return callback();
+				}
 				return;
 			}
 
-			if ( 'scrollBehavior' in document.documentElement.style ) {
-				el.scrollIntoView( { behavior: 'smooth' } );
-			} else {
-				el.scrollIntoView();
+			// For iOS Safari compatibility, use JS to set the minimum height.
+			var extraInfoArea = container.querySelector( '.jp-carousel-info-extra' );
+			if ( extraInfoArea ) {
+				extraInfoArea.style.minHeight = window.innerHeight - 64 + 'px';
+			}
+			var extraInfoContentArea = container.querySelector( '.jp-carousel-info-content-wrapper' );
+			var extraInfoContentAreaHeight = extraInfoContentArea
+				? extraInfoContentArea.offsetHeight
+				: window.innerHeight;
+
+			var isScrolling = true;
+			var startTime = Date.now();
+			var duration = 500;
```

Comment:
500ms feels a bit slow to me, I think 300ms is better and feels snappier, especially in places where the height of the toggled area is smaller.

Reply:
Oh, good feedback, I was wondering how folks might feel about the duration. I'll update to 300ms. Thanks!

Reply:
Updated to 300ms 👍